src/components/app-wrapper/app-wrapper.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I think the wrapper component isn't particularly demoing anything at the moment. @andrewjtait, it might be worth thinking about displaying other components inside it etc.

Reply:
I think it is a regular component.
https://carbon.sage.com/components/app-wrapper